editor/src/components/canvas/canvas-strategies/strategies/reparent-to-flex-strategy.tsx

```diff
@@ -75,12 +83,14 @@ export function baseAbsoluteReparentToFlexStrategy(
       ],
       fitness: fitness,
       apply: () => {
-        return applyFlexReparent(
-          'strip-absolute-props',
-          canvasState,
-          interactionSession,
-          reparentTarget,
-        )
+        return interactionSession == null
```

Comment:
This check should be unnecessary as we've already ensured that `interactionSession` is not null earlier on in the factory

Reply:
The source of mistake is that I started from the flex-reparent-to-flex strategy, and that did not use filteredSelectedElements at all